checker/tests/nullness/Issue1847B.java

```diff
@@ -0,0 +1,33 @@
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.function.Function;
+import org.checkerframework.checker.nullness.qual.KeyFor;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class Issue1847B {
```

Comment:
The two tests look very similar. Maybe a comment could highlight what the difference is?